UnityProject/Assets/Scripts/Health/IFireExposable.cs

```diff
@@ -0,0 +1,13 @@
+/// <summary>
```

Comment:
Not sure if this should be in the Health folder. Maybe a dedicated folder for fire?

Reply:
Same with all of the other fire scripts